src/parser.ts

```diff
@@ -30,10 +30,12 @@ const EXECUTION_TYPES: Record<StatementType, ExecutionType> = {
   UPDATE: 'MODIFICATION',
   CREATE_DATABASE: 'MODIFICATION',
   CREATE_TABLE: 'MODIFICATION',
+  CREATE_VIEW: 'MODIFICATION',
   CREATE_TRIGGER: 'MODIFICATION',
   CREATE_FUNCTION: 'MODIFICATION',
   DROP_DATABASE: 'MODIFICATION',
   DROP_TABLE: 'MODIFICATION',
+  DROP_VIEW: 'MODIFICATION',
```

Comment:
I think we need to start distinguishing between data modification and ddl modification soon.

Reply:
I would be onboard with this as it would allow for making more meaningful UI messages for DDL modifications where the number of "altered rows" is 0.

A v3 change to modify these all to be `DDL_MODIFICATION` vs the `MODIFICATION` that remains for `INSERT`, `UPDATE`, and `DELETE`.

Reply:
Yeah I like that a lot. Once I get time, I want to add support for ALTER TABLE, so this would pair nicely with that.